### ISSUE TITLE
Version confusion

### ISSUE THREAD

**Issue by beruic** [encode/django-rest-framework#5742]
What's the deal with versions 3.7.5, 3.7.6 and 3.7.7?
I am confused which one I should pull from PyPI, 3.7.4 being the well documented point release, and the others being, well, newer.

**Comment by rpkilby** [comment]
Hi @beruic. You should install the latest 3.7.x release. These releases fix a few packaging issues in 3.7.4

**Comment by rpkilby** [comment]
@carltongibson it might be beneficial to convert the 3.7.7 tag into a GH release that explains this.

**Comment by carltongibson** [comment]
And done. 🎉

**Comment by xordoquy** [comment]
Sad, http://www.django-rest-framework.org/topics/release-notes/#37x-series should cover it :)

**Comment by beruic** [comment]
Thank you :)

@xordoquy Well, the GitHub release page is usually for automatic change tracking and reporting through its atom feed. See https://github.com/encode/django-rest-framework/releases.atom. Therefore it is beneficial for a project to make good updates of this page. That said I see no problem in linking to the release notes in the tag message. I actually went looking for the information on http://www.django-rest-framework.org/ first, but since my eyes hit the GitHub button first, that's where I went, because, well, "GitHub! :D".
